examples/http-interceptor-error-handling/http_interceptor_error_handling.md

```diff
@@ -1,6 +1,6 @@
 # HTTP service - Interceptor error handling
 
-Errors that occurred in the request-response pipeline can be intercepted and handled by `ResponseErrorInterceptors`. In addition, a `RequestErrorInterceptor`  can be used to handle the errors that occurred in the request interceptor execution path. The `RequestErrorInterceptor` can send a response message according to the  error just like a `ResponseErrorInterceptor`. Moreover, it can modify the  request and dipatch it to the target service.
+Errors that occurred in the request-response flow can be intercepted and handled by `ResponseErrorInterceptors`. In addition, the `http:RequestErrorInterceptor` can be used to handle errors that occurred while executing `RequestInterceptors`. This error interceptor can send a response according to the error, just like a `ResponseErrorInterceptor`. Moreover, it can modify the request and dispatch it to the target service. Use `RequestErrorInterceptors` along with `RequestInterceptors` to validate the request beforehand and handle any errors during this validation.
```

Comment:
```suggestion
Errors that occurred in the request-response flow can be intercepted and handled by the `ResponseErrorInterceptors`. In addition, the `http:RequestErrorInterceptor` can be used to handle errors that occurred while executing the `RequestInterceptors`. This error interceptor can send a response according to the error similar to a `ResponseErrorInterceptor`. Moreover, it can modify the request and dispatch it to the target service. Use `RequestErrorInterceptors` along with `RequestInterceptors` to validate the request beforehand and handle any errors during this validation.
```